test/e2e/templates/java/javabuilddc.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Define a bunch of functions and set a bunch of variables
+TEST_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"` | grep -o '.*/oshinko-s2i/test/e2e')
+source $TEST_DIR/common
+
+SCRIPT_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"`)
+source $SCRIPT_DIR/../builddc
+
+JAVATEMP_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"` | grep -o '.*/oshinko-s2i/')/java
+set_template $JAVATEMP_DIR/javabuilddc.json
+set_git_uri https://github.com/radanalyticsio/jgrafzahl
+set_worker_count $S2I_TEST_WORKERS
+set_fixed_app_name java-build
+set_app_main_class io.radanalytics.jgrafzahl.App
+
+# Clear these
+set_app_file
+set_exit_flag
+
+function test_exit {
```

Comment:
hmm, this is supposed to be replaced with "test_fixed_exit" to eliminate the dup